lib/casino/ldap_authenticator.rb

```diff
@@ -77,17 +77,16 @@ def user_filter(username)
   end
 
   def extra_attributes(user_plain)
-    if @options[:extra_attributes]
```

Comment:
I was able to remove it without any specs failing, but that doesn't mean it's safe to remove. It turns out that line 52 already requires extra_attributes to be present. `@options[:extra_attributes].values` will give you NoMethodError: undefined method `values' for nil:NilClass if extra_attributes isn't included in the config.